.github/workflows/tot.yml

```diff
@@ -0,0 +1,75 @@
+name: run-tot-checks
+# Checks Puppeteer against the latest ToT build of Chromium.
+# Declare default permissions as read only.
+permissions: read-all
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # Supposed to be every day at 8 am (presumably UTC?).
```

Comment:
It _is_ UTC per https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#schedule